src/Driver.cpp

```diff
@@ -86,7 +86,7 @@ string Driver::binary_com(char const* str, size_t str_size)
 Driver::Driver(int max_packet_size, bool extract_last)
     : internal_buffer(new uint8_t[max_packet_size]), internal_buffer_size(0)
     , MAX_PACKET_SIZE(max_packet_size)
-    , m_stream(0), m_auto_close(true), m_extract_last(extract_last) 
```

Comment:
Avoid doing widespread space changes in an unrelated commit ... Leave it, I'll fix it before I merge.